modules/page/html/render/private.js

```diff
@@ -19,7 +20,10 @@ exports.create = function (api) {
     const i18n = api.intl.sync.i18n
     var id = api.keys.sync.id()
     var compose = api.message.html.compose({
-      meta: {type: 'post'},
+      meta: Struct({
+        type: 'post',
+        location: `${path}/${id}`
```

Comment:
The `id` in this case is actually the public key of the current user.

It potentially could be useful to include this, that way if later we add the ability to switch identities, drafts would be stored per identity. So you probably want to add the user's public key to all draft locations, or none, instead of just some.

Reply:
Seems to make sense to include it, I added it to standardize how it looks across the different views . So mainly for the sake of clarity in the code. However, you're right, it currently makes no difference if this is included. Good to see a future use case.